samples/subsys/ipc/openamp/sysbuild.cmake

```diff
@@ -9,9 +9,13 @@ ExternalZephyrProject_Add(
     BOARD ${SB_CONFIG_OPENAMP_REMOTE_BOARD}
   )
 
-# Add a dependency so that the remote sample will be built and flashed first
+# Add dependencies so that the remote sample will be built first
 # This is required because some primary cores need information from the
 # remote core's build, such as the output image's LMA
 add_dependencies(openamp openamp_remote)
```

Comment:
Is this line still needed?

Reply:
I guess so, if it's meant to control the order in which those images are built (not configured).